src/attributes.ts

```diff
@@ -625,6 +625,10 @@ const ATTRIBUTES_DEFINITION_BASE = {
 		name: 'Ankle_cuffs_chainable',
 		description: 'A cuff that is chainable to another item',
 	},
+	Handcover: {
+		name: 'Handcover',
+		description: 'A restraint that covers the wearer\'s hands',
```

Comment:
I suggest changing `A restraint` to `An item`, as nails for instance are no restraints.

Reply:
The nails don't have that attribute, but the gloves will, so you are correct here.